src/ui/public/vis/__tests__/_agg_config.js

```diff
@@ -5,23 +5,22 @@ import { VisProvider } from 'ui/vis';
 import { AggTypesAggTypeProvider } from 'ui/agg_types/agg_type';
 import { VisAggConfigProvider } from 'ui/vis/agg_config';
 import FixturesStubbedLogstashIndexPatternProvider from 'fixtures/stubbed_logstash_index_pattern';
-import { RegistryFieldFormatsProvider } from 'ui/registry/field_formats';
+import { fieldFormats } from 'ui/registry/field_formats';
```

Comment:
Just use `import { fieldFormats as fieldFormat }` then we can get rid of the `const fieldFormat = fieldFormats` in this file ... (or use search and replace :D)